modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/block/BlockService.scala

```diff
@@ -67,6 +59,24 @@ object BlockService {
 
         def getCollateral: Amount = collateral
       }
+
+      private def processAcceptanceSuccess(hashedBlock: Hashed[DAGBlock])(contextUpdate: BlockAcceptanceContextUpdate) = for {
+        hashedTransactions <- hashedBlock.signed.transactions.toNonEmptyList
+          .sortBy(_.ordinal)
+          .traverse(_.toHashed)
+
+        _ <- hashedTransactions.traverse(transactionStorage.accept)
+        _ <- blockStorage.accept(hashedBlock)
+        _ <- addressStorage.updateBalances(contextUpdate.balances)
+        isDependent = (block: Signed[DAGBlock]) => BlockRelations.dependsOn(Set(hashedBlock), Set.empty)(block)
+        _ <- blockStorage.restoreDependent(isDependent)
+      } yield ()
+
+      private def processAcceptanceError(hashedBlock: Hashed[DAGBlock])(reason: BlockNotAcceptedReason) =
+        blockStorage
+          .postpone(hashedBlock)
+          .map(_ => BlockAcceptanceError(BlockReference(hashedBlock.height, hashedBlock.proofsHash), reason))
```

Comment:
`.map(_ => x)` is equivalent to `.as(x)`

Reply:
Changed.